lms/djangoapps/course_api/views.py

```diff
@@ -170,4 +173,6 @@ def get_queryset(self):
         Return a list of courses visible to the user.
         """
         username = self.request.query_params.get('username', self.request.user.username)
-        return list_courses(self.request, username)
+        org = self.request.query_params.get('org')
```

Comment:
From the [DRF docs](http://www.django-rest-framework.org/api-guide/requests/#query_params):

> `request.query_params` is a more correctly named synonym for request.GET.